### ISSUE TITLE
New Vignette

### ISSUE THREAD

**Issue by mpadge** [njtierney/maxcovr#55]
How about killings lotsa proverbial birds with a stone or two, and writing a new vignette that serves as a submission for @richardbeare's [Geospatial and Transport Modelling in Stroke Service Planning](https://www.frontiersin.org/research-topics/8362/geospatial-and-transport-modeling-in-stroke-service-planning)? The way I see it, we'd just need to combine `osmdata`, `dodgr`, and `maxcovr` and the job would be done.

@njtierney: Would you be vaguely keen to pursue this?
- [ ] Yes
- [ ] No

@richardbeare: Does this sound like a workable submission from your side?
- [ ] Yes
- [ ] No

**Comment by richardbeare** [comment]
Yes, it sounds like a good start. The focus can be on both the types of geospatial data available via these packages and osm, as well as the general purpose computational components that are included. We'll probably need to tweak a little for the special issue, but most of the core will be the same.

**Comment by njtierney** [comment]
Yes!

This is something that I've been meaning to get back to @richardbeare about, I would love to collaborate to work on this article - do we have a specific topic in mind? Would it be easier to move this conversation to another repo or in email?
